### PR TITLE
CORDA-4861 - Update REPO readme to include the logo

### PR DESCRIPTION
**Pull request by fenryka**

### CODE REVIEW

**Review by lankydan — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -1,6 +1,16 @@
-# Corda Runtime OS
+<p align="center">
+  <img src="https://www.corda.net/wp-content/themes/corda/assets/images/crda-logo-big.svg" alt="Corda" width="500">
+</p>
+
+<a href="https://ci-master.corda.r3cev.com/viewType.html?buildTypeId=Corda_CordaBuild&tab=buildTypeStatusDiv&guest=1">
```
Comment (wzur-r3) [inline README.md:5]: this surely wrong - we do not use that CI server any longer

File: README.md [hunk 2]
```diff
@@ -1,6 +1,16 @@
...
+    <img src="https://ci.corda. r3cev.com/app/rest/builds/buildType:Corda_CordaBuild/statusIcon"/>
+</a> 
+
+[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0) [![Build Status](https://ci02.dev.r3.com/buildStatus/icon?job=Corda5%2Fcorda-runtime-os%2Frelease%252Fent%252F5.0)](https://ci02.dev.r3.com/job/Corda5/job/corda-runtime-os/job/release%252Fos%252F5.0/)
```
Comment (ronanbrowne) [inline README.md:9]: You'll have a merge conflict here as i just merged a few min ago here to update the badge URL.

Comment (fenryka) [reply README.md:9]: I'll pull your changes

**Review by ronanbrowne — APPROVED** [review]
LGTM

**Pull request merged**